pkg/store/store.go

```diff
@@ -52,8 +54,30 @@ func NewOpenTSDBStore(logger log.Logger, client opentsdb.ClientContext, reg prom
 		blockedMetricNames:      blockedMetricNames,
 		healthcheckMetric:       healthcheckMetric,
 	}
+	err := store.populateMaps()
+	if err != nil {
+		return nil, err
+	}
 	store.updateMetrics(context.Background(), logger)
-	return store
+	return store, nil
+}
+
+func (store *OpenTSDBStore) populateMaps() error {
```

Comment:
Given this is not dependent on external data I'd consider just doing this once in init, then the code doesn't have to pass errors around (just panic, as it will run on startup).